lib/Styled/Text.jsx

```diff
@@ -48,7 +48,7 @@ export const Text = styled.div`
     font-size: 15px;
   `}
   ${props =>
-    props.title &&
+    props.heading &&
```

Comment:
@reginapramesti have a look at what I've done in
https://github.com/TerriaJS/terrace/blob/master/lib/Styled/Heading.tsx#L20

Maybe it's time we brought that across, it's basically a wrapper around `Text` but with the proper html element for whichever heading we want to use, I'm not sure what the final solution/mishmash is going to look like, let me know what you think 🙂

Reply:
@soyarsauce oooh yeah that would be really good to have I reckon!